packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -220,6 +225,19 @@ class CreateCommand extends CreateBase {
     } else if (platforms == null || platforms.isEmpty) {
       throwToolExit('Must specify at least one platform using --platforms',
         exitCode: 2);
+    } else if (generateFfiPlugin &&
+        argResults.wasParsed('platforms') &&
+        platforms.contains('web')) {
+      throwToolExit('The web platform is not supported in plugin_ffi template.',
+          exitCode: 2);
+    } else if (generateFfiPlugin && argResults.wasParsed('ios-language')) {
+      throwToolExit(
+          'The ios-language is always C or C++ in plugin_ffi template.',
```

Comment:
For this and `android-language` can you explicitly call out that "using --template=plugin_ffi and ios-language" is not supported. Maybe: `'The option "ios-language" is not supported with the plugin_ffi template: the language will always be C or C++.'`